script.js

```
@@ -0,0 +1,159 @@
+
+        switch (scenario) {
+            case 'cmhc-bau':
+                // "Significantly faster than incomes" - let's assume a 5.5% annual growth.
+                annualGrowthRate = 0.055;
+                break;
+            case 'cmhc-is':
+                // "Balanced market" - let's assume a more modest 1.8% annual growth.
+                annualGrowthRate = 0.018;
+                break;
+            case 'imf':
+                const popGrowth = parseFloat(popGrowthSlider.value);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)